# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/x86_64-umount2.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -urN dietlibc-0.30/x86_64/dyn_syscalls.S dietlibc-0.30-t2/x86_64/dyn_syscalls.S
--- dietlibc-0.30/x86_64/dyn_syscalls.S	2005-10-08 23:11:32.000000000 +0200
+++ dietlibc-0.30-t2/x86_64/dyn_syscalls.S	2006-10-02 11:06:30.000000000 +0200
@@ -226,6 +226,7 @@
 #include "../syscalls.s/write.S"
 #include "../syscalls.s/writev.S"
 #include "../syscalls.s/ustat.S"
+#include "../syscalls.s/umount2.S"
 
 #include "../syscalls.s/clock_settime.S"
 #include "../syscalls.s/clock_gettime.S"
